cockatrice/src/shortcutssettings.h

```diff
@@ -157,214 +160,215 @@ class ShortcutsSettings : public QObject
-        {"Player/aIncP", ShortcutKey(QT_TRANSLATE_NOOP("shortcutsTab", "Add power (+1/+0)"),
+        {"Player/aIncP", ShortcutKey(QT_TRANSLATE_NOOP("shortcutsTab", "Add Power (+1/+0)"),
                                      parseSequenceString("Ctrl++"),
                                      ShortcutGroup::Power_Toughness)},
-        {"Player/aDecP", ShortcutKey(QT_TRANSLATE_NOOP("shortcutsTab", "Remove power (-1/-0)"),
+        {"Player/aDecP", ShortcutKey(QT_TRANSLATE_NOOP("shortcutsTab", "Remove Power (-1/-0)"),
                                      parseSequenceString("Ctrl+-"),
                                      ShortcutGroup::Power_Toughness)},
-        {"Player/aFlowP", ShortcutKey(QT_TRANSLATE_NOOP("shortcutsTab", "Move toughness to power (+1/-1)"),
+        {"Player/aFlowP", ShortcutKey(QT_TRANSLATE_NOOP("shortcutsTab", "Move Toughness to Power (+1/-1)"),
                                       parseSequenceString(""),
                                       ShortcutGroup::Power_Toughness)},
-        {"Player/aIncT", ShortcutKey(QT_TRANSLATE_NOOP("shortcutsTab", "Add toughness (+0/+1)"),
+        {"Player/aIncT", ShortcutKey(QT_TRANSLATE_NOOP("shortcutsTab", "Add Toughness (+0/+1)"),
                                      parseSequenceString("Alt++"),
                                      ShortcutGroup::Power_Toughness)},
-        {"Player/aDecT", ShortcutKey(QT_TRANSLATE_NOOP("shortcutsTab", "Remove toughness (-0/-1)"),
+        {"Player/aDecT", ShortcutKey(QT_TRANSLATE_NOOP("shortcutsTab", "Remove Toughness (-0/-1)"),
                                      parseSequenceString("Alt+-"),
                                      ShortcutGroup::Power_Toughness)},
-        {"Player/aFlowT", ShortcutKey(QT_TRANSLATE_NOOP("shortcutsTab", "Move power to toughness (-1/+1)"),
+        {"Player/aFlowT", ShortcutKey(QT_TRANSLATE_NOOP("shortcutsTab", "Move Power to Toughness (-1/+1)"),
                                       parseSequenceString(""),
                                       ShortcutGroup::Power_Toughness)},
-        {"Player/aIncPT", ShortcutKey(QT_TRANSLATE_NOOP("shortcutsTab", "Add power and toughness (+1/+1)"),
+        {"Player/aIncPT", ShortcutKey(QT_TRANSLATE_NOOP("shortcutsTab", "Add Power and Toughness (+1/+1)"),
                                       parseSequenceString("Ctrl+Alt++"),
                                       ShortcutGroup::Power_Toughness)},
-        {"Player/aDecPT", ShortcutKey(QT_TRANSLATE_NOOP("shortcutsTab", "Remove power and toughness (-1/-1)"),
+        {"Player/aDecPT", ShortcutKey(QT_TRANSLATE_NOOP("shortcutsTab", "Remove Power and Toughness (-1/-1)"),
                                       parseSequenceString("Ctrl+Alt+-"),
                                       ShortcutGroup::Power_Toughness)},
-        {"Player/aSetPT", ShortcutKey(QT_TRANSLATE_NOOP("shortcutsTab", "Set power and toughness"),
+        {"Player/aSetPT", ShortcutKey(QT_TRANSLATE_NOOP("shortcutsTab", "Set Power and Toughness"),
```

Comment:
`Set Power and Toughness...`

Reply:
I'll add the dots now, but I'm not calling it a blocker to test them all at the moment

Reply:
Collected and connected all the dots... :)